Assets/01_Scripts/ScoreText.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class ScoreText : MonoBehaviour
+{
+    private const int MaxAmountOfDigits = 10;
+    private const char LeftZeroChar = '0';
```

Comment:
The meaning behind the name of this var is not clear. Can you change it? 
By the way, can we use another writing convention for const variables, like, all in caps? Just for readability sake.